content/_code-samples/use-tickets/py/use-tickets-multisig.py

```diff
@@ -0,0 +1,133 @@
+from xrpl.models.transactions import TicketCreate, AccountSet, SignerListSet, SignerEntry
+from xrpl.transaction import safe_sign_and_submit_transaction, autofill, multisign, sign
+from xrpl.models.requests.account_objects import AccountObjects, AccountObjectType
+from xrpl.models.requests.submit_multisigned import SubmitMultisigned
+from xrpl.wallet import generate_faucet_wallet
+from xrpl.clients import JsonRpcClient
+
+# This code sample lists which tickets are outstanding against an account
+# and use the ticket for a multisignature transaction.
+# A funded account on the testnet is provided for testing purposes
```

Comment:
Explaining why you'd want to do this and removing an unnecessary line about using a testnet account.
```suggestion
# This code sample shows how to use tickets with multisigning
# to allow you to do a series of transactions in any order while you wait
# for signers to come back with their signatures.
```